lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -105,11 +124,16 @@ class TagsBlockingFirstPaint extends Gatherer {
       return tags.reduce((prev, tag) => {
         const request = requests[tag.url];
         if (request && !request.isLinkPreload) {
+          const nonMatchingMediaChangeTimes = (tag.mediaChanges || [])
```

Comment:
add a few comments to describe why we need to do this

Reply:
done